Documentation/Upgrade/InstallTheNewSource/Index.rst

```diff
@@ -21,29 +21,29 @@ Upgrading a Composer Based System
 
 .. caution::
 
-   TYPO3 v9 LTS has no support for the package `typo3/cms` anymore - instead
+   TYPO3 v10 LTS has no support for the package `typo3/cms` anymore - instead
    all TYPO3 system extensions are now single packages. When upgrading, remove
    the `typo3/cms` package and require the single packages you need. To make
    selecting the packages easier, you can use
    https://get.typo3.org/misc/composer/helper to find the packages you need.
 
 Use composer to update your system via for example::
 
-   composer require typo3/cms-backend:^9.5 typo3/cms-core:^9.5 \
-      typo3/cms-extbase:^9.5 typo3/cms-extensionmanager:^9.5 \
-      typo3/cms-filelist:^9.5 typo3/cms-fluid:^9.5 typo3/cms-frontend:^9.5 \
-      typo3/cms-install:^9.5 typo3/cms-recordlist:^9.5 \
+   composer require typo3/cms-backend:^10.4 typo3/cms-core:^10.4 \
+      typo3/cms-extbase:^10.4 typo3/cms-extensionmanager:^10.4 \
+      typo3/cms-filelist:^10.4 typo3/cms-fluid:^10.4 typo3/cms-frontend:^10.4 \
+      typo3/cms-install:^10.4 typo3/cms-recordlist:^10.4 \
       --update-with-dependencies
 
 If you have extensions installed, you will have to find the corresponding newer
 versions you want to install for your new major version and update them, too.
 
 Example::
 
-   composer require typo3/cms-backend:^9.5 typo3/cms-core:^9.5 \
-      typo3/cms-extbase:^9.5 typo3/cms-extensionmanager:^9.5 \
-      typo3/cms-filelist:^9.5 typo3/cms-fluid:^9.5 typo3/cms-frontend:^9.5 \
-      typo3/cms-install:^9.5 typo3/cms-recordlist:^9.5 georgringer/news:^7.0 \
+   composer require typo3/cms-backend:^10.4 typo3/cms-core:^10.4 \
+      typo3/cms-extbase:^10.4 typo3/cms-extensionmanager:^10.4 \
+      typo3/cms-filelist:^10.4 typo3/cms-fluid:^10.4 typo3/cms-frontend:^10.4 \
+      typo3/cms-install:^10.4 typo3/cms-recordlist:^10.4 georgringer/news:^7.0 \
```

Comment:
`georgringer/news:^8.0` 

I would suggest to add a link to where to get that information if an extension must be updated and to which minimal version (https://packagist.org/packages/georgringer/news#7.3.1)

Reply:
Indeed, news version is updated now.